notebooks/responsible_ai/explainable_ai/solutions/xai_text_with_LIT.ipynb

```diff
@@ -0,0 +1,2770 @@
+{
```

Comment:
<p>Can you add a quiet install on the libraries? There's a lot of printed statements</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/527/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Done!</p>